templates/pma.config.mustache

```diff
@@ -30,7 +30,7 @@ $i++;
 /* Authentication type */
 $cfg['Servers'][$i]['auth_type'] = 'cookie';
 /* Server parameters */
-$cfg['Servers'][$i]['host'] = 'db';
+$cfg['Servers'][$i]['host'] = 'global-db';
```

Comment:
Yes. This is the configuration of what will be the most commonly used host. If the hostname is not entered this will be taken. So it is set to `global-db`. This will work for local as well as remote dbs too. It is just that in those special cases they will have to give hostname in login. 

>should it be 'db' if it's local db and 'global-db' if it's global?
No. This is host mounted file common for all sites. It does not depend on any particular site. It can only have the most commonly used defaults as discusssd above.

Reply:
Got it. Thanks for the detailed explaination